pokedex/data/csv/pokemon_species_flavor_text.csv

```diff
@@ -188199,3 +188199,17 @@ Pokémon has raised since the child’s infancy."
 893,34,12,"长在身上的藤蔓断掉之后
 会变成土壤的养分，
 帮助森林中的植物成长。"
+894,33,9,"This Pokémon is a cluster of electrical energy. It's said that removing the rings on Regieleki's body will unleash the Pokémon's latent power."
+894,34,9,"Its entire body is made up of a single organ that generates electrical energy. Regieleki is capable of creating all Galar's electricity."
+895,33,9,"An academic theory proposes that Regidrago's arms were once the head of an ancient dragon Pokémon. The theory remains unproven."
+895,34,9,"Its body is composed of crystallized dragon energy. Regidrago is said to have the powers of every dragon Pokémon."
+896,33,9,"Glastrier emits intense cold from its hooves. It's also a belligerent Pokémon—anything it wants, it takes by force."
+896,34,9,"Glastrier has tremendous physical strength, and the mask of ice covering its face is 100 times harder than diamond."
+897,33,9,"It probes its surroundings with all its senses save one—it doesn't use its sense of sight. Spectrier's kicks are said to separate soul from body."
+897,34,9,"As it dashes through the night, Spectrier absorbs the life-force of sleeping creatures. It craves silence and solitude."
+898,33,9,"Calyrex is a merciful Pokémon, capable of providing healing and blessings. It reigned over the Galar region in times of yore."
+898,34,9,"Calyrex is known in legend as a king that ruled over Galar in ancient times. It has the power to cause hearts to mend and plants to spring forth."
+20000,33,9,"According to lore, this Pokémon showed no mercy to those who got in its way, yet it would heal its opponents' wounds after battle."
```

Comment:
Same here, let's stick with the standard pokemon, so no variants.

Reply:
please, get rid of the 2 lines.

Reply:
Here :)

Reply:
Fixed it! Thanks. Removed the two Shadow/Ice Rider Calyrex form descriptions fro m the species flavor text.